waspc/src/Wasp/Generator/SdkGenerator/AuthG.hs

```diff
@@ -30,7 +30,7 @@ genAuth spec =
     Just auth ->
       -- shared stuff
       sequence
-        [ genUserTs
+        [ genFileCopy [relfile|auth/user.ts|]
```

Comment:
No need for templating anymore for this file, I've used a copy file draft instead.